apps/expo/src/components/handle-availability.tsx

```diff
@@ -53,21 +57,27 @@ export const HandleAvailabilityResult = ({
         return (
           <View className="flex-row items-center">
             <CheckCircle2Icon className="mr-1.5 text-green-700" size={14} />
-            <Text className="text-sm text-green-700">Available</Text>
+            <Text className="text-sm text-green-700">
+              <Trans>Available</Trans>
+            </Text>
           </View>
         );
       case "taken":
         return (
           <View className="flex-row items-center">
             <XCircleIcon className="mr-1.5 text-red-500" size={14} />
-            <Text className="text-sm text-red-500">Handle is taken</Text>
+            <Text className="text-sm text-red-500">
+              <Trans>Handle is taken</Trans>
```

Comment:
What about changing this to "Handle already taken" to match one of the strings in change-handle.tsx?